### PR TITLE
Add free plan article

### PR DESCRIPTION
**Pull request by staceysalamon-aiven**
# What changed, and why it matters
Information on free plan services.

### CODE REVIEW

File: _toc.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ entries:
       - file: docs/platform/concepts/aiven-node-firewall-configuration
       - file: docs/platform/concepts/authentication-tokens
       - file: docs/platform/concepts/availability-zones
+      - file: docs/platform/concepts/free-plan
```
Comment (carabaestlein) [inline _toc.yml:16]: Should the title be 'Free plans'?

**Review by carabaestlein — COMMENTED** [review]
Left one small comment, other than that LGTM!

File: _toc.yml (hunk 1 above)
Comment (staceysalamon-aiven) [reply _toc.yml:16]: Or "Free plan services" maybe?

Comment (carabaestlein) [reply _toc.yml:16]: Let's leave it as is given we have already put this link in quite a few places :)

**Review by carabaestlein — APPROVED** [review]

**Pull request merged**